mozilla_bitbar_devicepool/configuration.py

```diff
@@ -65,17 +75,66 @@ def configure(bitbar_configpath, filespath=None, update_bitbar=False):
 
     FILESPATH=filespath
 
-    with open(bitbar_configpath) as bitbar_configfile:
-        CONFIG = yaml.load(bitbar_configfile.read(), Loader=yaml.SafeLoader)
-
     logger.info('configure: starting configuration')
     start = time.time()
+
+    with open(bitbar_configpath) as bitbar_configfile:
+        CONFIG = yaml.load(bitbar_configfile.read(), Loader=yaml.SafeLoader)
+    expand_configuration()
+    if update_bitbar:
```

Comment:
Should this explode even if we're not updating bitbar?

Reply:
I can see arguing for both sides of this. I think having it alert that there is a problem even if we aren't updating would be best. What do you think?

Reply:
I replied in the main UI, but to comment here as well, I think so yes but am open to not doing so if you wish.

Reply:
Changed to always run.